modules/raft/src/main/java/org/apache/ignite/internal/raft/Loza.java

```diff
@@ -100,9 +104,11 @@ public class Loza implements IgniteComponent {
      * @param dataPath      Data path.
      * @param clock         A hybrid logical clock.
      */
-    public Loza(ClusterService clusterNetSvc, Path dataPath, HybridClock clock) {
+    public Loza(ClusterService clusterNetSvc, ReplicaManager replicaMgr, Path dataPath, HybridClock clock) {
```

Comment:
Loza should know nothing about replicaManager.

Reply:
Fixed.